src/presentation/react/core/components/sync-summary/SyncSummary.tsx

```diff
@@ -246,7 +256,6 @@ const SyncSummary = ({ response, onClose }: SyncSummaryProps) => {
                             <AccordionDetails className={classes.accordionDetails}>
                                 <Typography variant="overline">{i18n.t("Summary")}</Typography>
                             </AccordionDetails>
-
                             {message && (
```

Comment:
Do not remove newline:

```suggestion

                            {message && (
```